src/main/java/org/syncninja/util/TwoFilesComparison.java

```diff
@@ -0,0 +1,43 @@
+package org.syncninja.util;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.*;
+
+public class TwoFilesComparison {
```

Comment:
Can pick a better name, FileCompareUtil for example, don't mention numbers in the name